flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/decorators/InitJobManagerDecorator.java

```diff
@@ -63,6 +64,9 @@ public FlinkPod decorateFlinkPod(FlinkPod flinkPod) {
 			.editOrNewMetadata()
 				.withLabels(kubernetesJobManagerParameters.getLabels())
 				.withAnnotations(kubernetesJobManagerParameters.getAnnotations())
+				.withOwnerReferences(kubernetesJobManagerParameters.getOwnerReference().stream()
```

Comment:
I am afraid that we could not set owner reference to the JobManager pod. I believe you mean to set the owner reference to the JobManager deployment.

Then this piece of codes should be moved into `KubernetesJobManagerFactory#createJobManagerDeployment`.

Reply:
oh yes, my bad

Reply:
Fixed